src/en/juju-concepts.md

```diff
@@ -1,31 +1,66 @@
-Title: Juju concepts
-TODO:  Review required
+Title: Concepts and terms
 
-# Juju concepts
+# Concepts and terms
 
-## The Juju client
+## Cloud
 
-The Juju client is command line interface (CLI) software that is used to
-manage Juju, whether as an administrator or as a regular user. It is 
-installed, via an APT package ('juju'), onto your personal workstation. 
-This software manages your connection to Juju controllers in the cloud, 
-and is used to issue commands to deploy and manage applications.
+To Juju, a *cloud* (or backing cloud) is a resource which provides machines
+(instances), and possibly storage, in order for application units to be
+deployed upon them. This includes public clouds such as Amazon Web Services,
+Google Compute Engine, and Microsoft Azure as well as private OpenStack-based
+clouds. Juju can also make use of environments which are not clouds per se, but
+which Juju can nonetheless treat as a cloud. [MAAS][maas] and [LXD][lxd] fit
+into this last category.
+
+See [Clouds][clouds] to learn more.
+
+## Controller
+
+The Juju *controller* is the initial cloud instance which is created in order
+for Juju to gain access to a cloud. It is created by having the Juju client
+contact the cloud's API. The controller is a central management node for the
+chosen cloud, taking care of all operations requested by the Juju client.
+Multiple clouds (and thus controllers) are possible and each one may contain
+multiple models and users.
+
+For more information see [Controllers][controllers].
+
+## Client
```

Comment:
I don't think this is the right place for client or agent - either put them both at the top or both at the bottom

Reply:
I put the client immediately after controller because the essence of the client lies in its connection to the controller.
Likewise, I wanted to put the agent as close as possible to the controller as it also connects to the controller. I put it after model because that concept brought in the idea of a machine (where the agent runs).